### DISCUSSION TITLE
【第四期飞桨黑客松】模型套件开源贡献——3D任务开发套件

### DISCUSSION THREAD

**Discussion by cloud2009** [PaddlePaddle/Paddle#50730] [General]
[PaddlePaddle Hackathon 第四期](https://github.com/PaddlePaddle/Paddle/issues/50629#3d)
大家可以在这里讨论有关3D任务开发套件的问题哦～

**Comment by weihongwei-zg** [comment]
详细描述：
将单目，BEV，点云检测结果可视化
完成复现后合入Paddle3D
参考repo https://github.com/open-mmlab/mmdetection3d/tree/master/demo
提交内容：
代码、文档，合入Paddle3D套件
技术要求：
熟练掌握Python、PaddlePaddle和3D目标检测算法

**Comment by weihongwei-zg** [comment]
第一次参加这个类型比赛，可以理解成是三个要求么？
1.单目视觉（monodet）可视化，这个在paddle3d的smoke算法中已经有vis.py文件了，还需要重复提交嘛？
2.单模态点云3d检测可视化，随便选个网络做推理就好了嘛（例如PointPillars）
3.mm3d里面的那个demo中似乎没有bev可视化
还有就是可视化代码要放到paddle3d的哪个地方，也是像mm3d这样建个demo文件夹么？

**Comment by zbdehh** [comment]
题号：174
问题：关于具体复现指标，是使用lego数据集在paper同样的config下在全量测试集上达到平均psnr达到32.54吗？

**Reply by will-jl944** [reply]
是的

**Reply by will-jl944** [reply]
代码最终需要合入Paddle的3D渲染repo，目前还没有开源，可以以这个[repo](https://github.com/will-jl944/PaddleRendering)作为参考，遵循这里的代码结构，并且里面的模块可以复用。

**Reply by zbdehh** [reply]
发现这个任务已经被锁定了呢，目前在你仓库内测试的结果是32.41.还可以锁定其它题目吗？

**Reply by will-jl944** [reply]
锁定题目需要提交简历并通过我们的筛选，以接到我们锁定题目的通知为准。目前除题目174以外的其他神经渲染类模型都还没有被锁定，可以提交简历申请。
